tgui/packages/tgui/interfaces/MiningVendor.js

```diff
@@ -48,7 +53,7 @@ export const MiningVendor = (props, context) => {
                         'min-width': '95px',
                         'text-align': 'center',
                       }}
-                      disabled={!data.user
+                      disabled={!data.user.access_valid
```

Comment:
add `|| data.user.observer`

Reply:
No. That'll enable button clicks for ghosts. (although they can't purchase even if the button is active)

Reply:
disabled if user is observer... it's flipped

Reply:
ah, right